webapp/lib/main_ui.dart

```diff
@@ -43,6 +43,15 @@ class CodaUI {
     fbt.init();
     auth.init();
     snackbar.init();
+
+    window.onError.listen((e) {
+      if (e is ErrorEvent) {
+        var error = e.error;
+        var stacktraceOrFilename = error is Error ? error.stackTrace : e.filename;
+        log.severe('${e.message}: ${stacktraceOrFilename}');
```

Comment:
Could we JSON encode this please? the equivalent of 
```
log.severe(json.dumps({
"messageType": "Error",
"message" : e.message,
"stackTrace" : error is Error ? error.stackTrace : null,
"filename" : error is Error ? null : e.filename
}))
```

Reply:
Thanks, that looks so much better. Done.